### PR TITLE
feat: Define a new lint script

### PR DESCRIPTION
**Pull request by Bouncey**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.

### CODE REVIEW

File: api-server/package.json [hunk 1]
```diff
@@ -91,9 +91,6 @@
     "babel-plugin-transform-imports": "^1.5.1",
     "commitizen": "^2.9.6",
     "cz-freecodecamp": "^1.0.1",
-    "eslint": "^4.10.0",
-    "eslint-config-freecodecamp": "^1.1.1",
```
Comment (ValeraS) [inline api-server/package.json]: do we not follow the freecodecamp rules anymore?

Comment (raisedadead) [reply api-server/package.json]: I think the intent is to add them to the root of the repo. I am all for moving all dev deps to the root.

Comment (ValeraS) [reply api-server/package.json]: But linting was done without these rules.

Comment (ValeraS) [reply api-server/package.json]: Ok, I was wrong and it should be using config from the root of the project, but I don't understand why did it insert trailing comma here https://github.com/freeCodeCamp/freeCodeCamp/pull/35076/files#diff-d4492df91628c2d780b70df6ad7944aaR21

Comment (Bouncey) [reply api-server/package.json]: That may have been my editor no picking up the `.prettierrc`.

**Comment by Bouncey** [comment]
I have placed this PR under a WIP as I would like to consolidate everything in to the project root, and have prettier working correctly. Currently ESlint is throwing in react files, which is just silly.

File: api-server/package.json (hunk 1 above)
Comment (Bouncey) [reply api-server/package.json]: I have swapped this out for an updated `@freecodecamp/eslint-config` package which brings the config up to date with the latest eslint/babel-eslint/plugins.

**Review by ValeraS — CHANGES_REQUESTED** [review]
Do we plan to use prettier with eslint?

[eslint-config-prettier, eslint-plugin-prettier](https://prettier.io/docs/en/eslint.html)

**Comment by Bouncey** [comment]
@ValeraS The idea is to implement prettier in some form after this PR. Should I just implement it now? Seeing as i am already touching a lot of files.

**Comment by Bouncey** [comment]
Now with `prettier-eslint`

**Comment by ValeraS** [comment]
@Bouncey @raisedadead could you look at https://github.com/ValeraS/freeCodeCamp/commits/feat/prettier

with this use of the prettier, I see only the one main difference from our rules (wrap-regex). What do you think, can we use it as the prettier suggests?

**Comment by Bouncey** [comment]
I have merged your changes in to this PR @ValeraS 

Thank you

**Review by ValeraS — APPROVED** [review]

**Pull request merged**